### ISSUE TITLE
Listing pages should have blank alt text

### ISSUE THREAD

**Issue by pollecuttn** [wellcomecollection/wellcomecollection.org#10268]
**What**

Cards use decorative images, not requiring alt text

**Where**

Listings pages e.g. https://wellcomecollection.org/events

**TODO**
- [ ] Make sure alt text is blank for decorative images on listings pages

**Background**

From the Hassell Inclusion accessibility audit August 2023.

Dovetail: https://proficient-wallaby-dpwd.dovetailapp.com/insights/Cards-use-decorative-images-not-requiring-alt-text-1OaCcelPmIswL44x2zikO

Miro board: https://miro.com/app/board/uXjVMjsL5YE=/
